sphinx_automodapi/tests/test_automodsumm.py

```diff
@@ -49,10 +49,17 @@ def teardown_function(func):
 
 
 def write_api_files_to_tmpdir(tmpdir):
-    apidir = tmpdir.mkdir('api')
-    with open(apidir.join('sphinx_automodapi.tests.example_module.mixed.add.rst').strpath, 'w') as f:
+    apidir = tmpdir.mkdir("api")
```

Comment:
Annoying. I'll revert the style improvements. My VSCode is configured to format on save. Normally saves time.